build-configs/BuildConfigType.ts

```diff
@@ -30,6 +30,8 @@ export type FeatureFlagsType = FixedCityType & {
   sentry: boolean
   // Enables error tracking to sentry, can be disabled by users.
   developerFriendly: boolean // Enables additional debugging output for devs (i18n, redux, hidden cities, version).
+  // Enables the option for users to suggest Integreat to their city
+  cityNotCooperating: boolean
```

Comment:
Reading this variable name I'd have no clue what it is about. I'd rename to something like `recommendToCityIfNotCooperating` or similar.